FF16Tools.Files/Timelines/Elements/General/TimelineElement_84.cs

```diff
@@ -0,0 +1,64 @@
+﻿using FF16Tools.Files.Timelines.Chara;
+
+using Syroot.BinaryData;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FF16Tools.Files.Timelines.Elements.General;
+
+public class TimelineElement_84 : TimelineElementBase, ISerializableStruct
+{
```

Comment:
you read/wrote everything so i belive it worked, but this looks very different than the template version  
https://github.com/Nenkai/010GameTemplates/blob/main/Square%20Enix/Final%20Fantasy%2016/FF16_tlb_FileCharaTimeLine.bt#L454

Reply:
Looks correct, after double checking